src/app/components/ThemeProvider/themes/loadableConfig.ts

```diff
@@ -140,9 +140,18 @@ export const themes = {
     () => import(/* webpackChunkName: "themes-ukrainian" */ './ukrainian'),
   ),
   urdu: loadable(() => import(/* webpackChunkName: "themes-urdu" */ './urdu')),
-  uzbek: loadable(
-    () => import(/* webpackChunkName: "themes-uzbek" */ './uzbek'),
-  ),
+  uzbek: {
+    // half refactored - not sure if it's a good idea to have multiple chunks with the same name
+    default: loadable(
+      () => import(/* webpackChunkName: "themes-uzbek-cyr" */ './uzbek/cyr'),
```

Comment:
I'm not sure if it's a good idea to have multiple chunks with the same name. I believe this helps to reduce bundle size?